src/frontend/src/components/Posts/Posts.jsx

```diff
@@ -11,6 +12,19 @@ const useStyles = makeStyles(() => ({
     padding: 0,
     maxWidth: '785px',
   },
+  error: {
+    position: 'center',
+    color: '#B5B5B5',
```

Comment:
@cindyledev, @agarcia-caicedo or someone else who knows our theme code: are there theme values we should be using for these colours instead?